.gitignore

```diff
@@ -9,3 +9,4 @@ lerna-*.log
 /packages/**/*.js.map
 /packages/**/*.d.ts
 .DS_Store
+dist
```

Comment:
In the future, all distributed assets will belong in a `dist` directory instead of being compiled in place.